### PR TITLE
FIX: pmpsdb_client didn't get updated

### PR DESCRIPTION
**Pull request by ZLLentz**
Update the deps and pull the ci yaml back into the branch for a new current-env

Incidentally picks up a shellcheck update.

Added the Following Packages
----------------------------

- flask-authnz
- pmpsdb_client
- pmpsui

Added the Following Dependencies
--------------------------------

- fabric (required by pmpsdb-client, pmpsdb_client)
- invoke (required by fabric, paramiko, which are used in bluesky, bluesky-live, databroker, hklpy, ophyd, ophyd-async, pandas, pcdsdevices, pmpsdb_client, scikit-image, suitcase-tiff, tiled, xarray)

Removed the Following Packages
------------------------------

- pmpsdb-client

### CODE REVIEW

File: envs/pcds/conda-packages.txt [hunk 1]
```diff
@@ -60,6 +60,7 @@ pcdswidgets>=0.8.3
 periodictable>=1.5.2
 pipdeptree>=2.18.1
 pmgr>=2.1.2
+pmpsdb_client>=1.2.0
```
Comment (tangkong) [inline envs/pcds/conda-packages.txt:63]: wait is there a conda-feedstock for this?  Or should it still be in pip-packages

Comment (ZLLentz) [reply envs/pcds/conda-packages.txt:63]: no conda-feedstock but it's on the pcds-tag channel

**Review by tangkong — APPROVED** [review]
moving forward I propose no python package ever use `-` or `_`, because this kind of nitpicking is awful

**Comment by ZLLentz** [comment]
You're right, the naming mixups are awful and I won't participate in it again. Definitely no `-` ever again, probably no `_` either and just go no-spaces like you chose with `pmpsui`.

**Pull request merged**